data/json/proficiencies/melee_weapons.json

```diff
@@ -0,0 +1,842 @@
+    "category": "prof_combat",
+    "name": { "str": "Flail Proficiency" },
+    "description": "You're able to swing your flail with deadly precision, crushing through ribs and skulls alike.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "16 h",
+    "required_proficiencies": [ "prof_flails_familiar" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_flails_master",
+    "category": "prof_combat",
+    "name": { "str": "Flail Mastery" },
+    "description": "You can make your flail dance like a ballerina, and hit like a truck.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "32 h",
+    "required_proficiencies": [ "prof_flails_pro" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_maces_familiar",
+    "category": "prof_combat",
+    "name": { "str": "Mace Familiarity" },
+    "description": "You've got the gist of bludgeoning your enemies.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_maces_pro",
+    "category": "prof_combat",
+    "name": { "str": "Mace Proficiency" },
+    "description": "You swing your mace with dead efficiency.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "16 h",
+    "required_proficiencies": [ "prof_maces_familiar" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_maces_master",
+    "category": "prof_combat",
+    "name": { "str": "Mace Mastery" },
+    "description": "Your strikes with a mace shatter skulls and pulp brains.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "32 h",
+    "required_proficiencies": [ "prof_maces_pro" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_great_swords_familiar",
+    "category": "prof_combat",
+    "name": { "str": "Great Sword Familiarity" },
+    "description": "You're not afraid to lift a heavy sword nor bring it down on someone's head.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_great_swords_pro",
+    "category": "prof_combat",
+    "name": { "str": "Great Sword Proficiency" },
+    "description": "You are not afraid to swing big with a heavy sword.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "16 h",
+    "required_proficiencies": [ "prof_great_swords_familiar" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_great_swords_master",
+    "category": "prof_combat",
+    "name": { "str": "Great Sword Mastery" },
+    "description": "You wield your blade like it was a modern day Caliburn.",
```

Comment:
Was Excalibur a two-hander tho 🤔

Reply:
I don't know, but it was a pretty great sword.